src/routes/dashboard/components/Dashboard.js

```diff
@@ -68,20 +68,39 @@ class Dashboard extends React.Component<PropsType> {
     ]
   }
 
-  landing = () => this.props.navigation.navigate('Landing')
+  landing = () => this.props.navigation.navigate('Landing');
```

Comment:
I guess you are using an indentation of 4 spaces while we are using 2 spaces. Please change this :) If you are using IntelliJ, you can find the setting here: Settings > Editor > Code Style > JavaScript > Punctuation